docs/CONSOLE_COLOR_CODES.md

```
@@ -0,0 +1,82 @@
+|---------------|---------------|
+| `§0` – `§9`  | Colors        |
+| `§a` – `§f`  | Colors        |
+| `§l`          | Bold          |
+| `§o`          | Italic        |
+| `§n`          | Underline     |
+| `§m`          | Strikethrough |
+| `§r`          | Reset         |
+
+Example raw line:
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)